build/local_builder.go

```diff
@@ -256,7 +257,7 @@ func (l *LocalBuilder) execLayer(layer *v1.Layer, tempTarget string) error {
 			dist = filepath.Join(tempTarget, strings.Fields(layer.Value)[1])
 		} else {
 			// src is file
-			dist = filepath.Join(tempTarget, strings.Fields(layer.Value)[1], strings.Fields(layer.Value)[0])
+			dist = filepath.Join(tempTarget, strings.Fields(layer.Value)[1], path.Base(strings.Fields(layer.Value)[0]))
```

Comment:
you could replace ‘path’ with ‘filepath’ to ensure our code will be matched with actual file system.

Reply:
by the way , you can comment your PR  /test build to run e2e test for build.